src/test/starknet.t.sol

```diff
@@ -127,8 +127,6 @@ contract StarknetTests is GoerliDssSpellTestBase, ConfigStarknet {
 
         assertEq(daiBridge.wards(addr.addr("MCD_PAUSE_PROXY")), 1, "StarknetTest/pause-proxy-not-ward-on-dai-bridge");
         assertEq(daiBridge.wards(addr.addr("MCD_ESM")),         1, "StarknetTest/esm-not-ward-on-dai-bridge");
-        // TODO verify that this is intentional or necessary
```

Comment:
I think I read something about this, but can I get a bit of better context, please?

Reply:
This removed testing line was backported from mainnet in the previous goerli spell work but was never archived or ran.
If it would have been ran it would have failed since the escrow mom was not authed on the goerli dai bridge at any point.
The escrow mom was authed on mainnet dai bridge but that has been removed/fixed in the last mainnet spell.

Reply:
Ahh ok got it.